core/ledger/kvledger/txmgmt/statedb/statecouchdb/cache.go

```diff
@@ -12,7 +12,8 @@ import (
 )
 
 var (
-	keySep = []byte{0x00}
+	keySep                  = []byte{0x00}
+	fastCacheValueSizeLimit = 64 * 1024
```

Comment:
Just to take a note, I am typically not comfortable in hardcoding these values that are dependent on the code of the underlying library. Because, no one will remember this before upgrading the library. However, in this case, unfortunately, this is not exported variable in the library. See a related comment below.